d2game/d2gamescreen/character_select.go

```diff
@@ -67,10 +67,16 @@ func CreateCharacterSelect(
 	ui *d2ui.UIManager,
 	connectionType d2clientconnectiontype.ClientConnectionType,
 	connectionHost string,
-) *CharacterSelect {
-	// https://github.com/OpenDiablo2/OpenDiablo2/issues/790
-	playerStateFactory, _ := d2hero.NewHeroStateFactory(asset)
-	entityFactory, _ := d2mapentity.NewMapEntityFactory(asset)
+) (*CharacterSelect, error) {
+	playerStateFactory, err := d2hero.NewHeroStateFactory(asset)
+	if err != nil {
+		return &CharacterSelect{}, err
```

Comment:
This should return nil, err. Same with entityfactory.

Reply:
Corrected this.